pkg/kubelet/status/status_manager.go

```diff
@@ -601,7 +601,6 @@ func (m *manager) syncPod(uid types.UID, status versionedPodStatus) {
 	// We don't handle graceful deletion of mirror pods.
```

Comment:
Please fix this comment, since apparently we now do :)

Reply:
thanks fixed.